GDCpp/GDCpp/IDE/AndroidExporter.h

```diff
@@ -9,7 +9,7 @@ class wxProgressDialog;
  * \brief Allow to export a game for Android
  * \todo Highly experimental
  *
- * To try it, replace the exporter in CppPlatform::GetProjectExporter and
+ * To try it, replace the exporter in CppPlatform::GetProjectExporters and
```

Comment:
@victorlevasseur I think this comment should be rewritten as the exporter is now directly available from the File menu. Also maybe add a link to the wiki article being written? :)